### PR TITLE
chore(volume): Add nil pointer checks and tests for outputResult func…

### PR DESCRIPTION
**Pull request by Fyusel**
fixes: #574

### CODE REVIEW

File: internal/cmd/beta/volume/list/list_test.go [hunk 1]
```diff
@@ -202,3 +201,30 @@ func TestBuildRequest(t *testing.T) {
...
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "empty",
+			args:    args{},
+			wantErr: false,
+		},
+	}
```
Comment (marceljk) [inline internal/cmd/beta/volume/list/list_test.go:220]: Can you add here a test case with an empty volume within the volumes slice?
```
		{
			name:    "set empty volume",
			args:    args{
				volumes: []iaas.Volume{{}},
			},
			wantErr: false,
		},
```

File: internal/cmd/beta/volume/performance-class/list/list_test.go [hunk 2]
```diff
@@ -202,3 +201,30 @@ func TestBuildRequest(t *testing.T) {
...
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "empty",
+			args:    args{},
+			wantErr: false,
+		},
```
Comment (marceljk) [inline internal/cmd/beta/volume/performance-class/list/list_test.go:219]: And also here

**Comment by marceljk** [comment]
https://github.com/stackitcloud/stackit-cli/blob/31084ad3c4e1446d0b07349dae7759d618e137da/internal/cmd/beta/volume/delete/delete.go#L59-L64
Could you add an additional check here if the volume name is empty? If it's empty, it should use the volumeId as label
This would solve this issue #574

**Review by marceljk — APPROVED** [review]
